envbuilder.go

```diff
@@ -196,6 +197,10 @@ type Options struct {
 	// This is optional!
 	GitPassword string `env:"GIT_PASSWORD"`
 
+	// GitHttpProxyURL is the url for the http proxy.
+	// This is optional!
+	GitHttpProxyURL string `env:"GIT_HTTP_PROXY_URL"`
```

Comment:
Minor style suggestion:

```suggestion
	GitHTTPProxyURL string `env:"GIT_HTTP_PROXY_URL"`
```

Initialisms in go should be all-caps: https://go.dev/wiki/CodeReviewComments#initialisms